### ISSUE TITLE
Use "confirm" instead of "verify" everywhere in the app

### ISSUE THREAD

**Issue by isabellewei** [celo-org/celo-monorepo#6887]
### Expected Behavior

User should see `confirm` or `confirmation` when referring to the verification process in Valora

### Current Behavior

Currently still have some references to `verify` and `verification` https://celo-org.slack.com/archives/C018VLA3YAK/p1612339201168700

**Comment by coreycelo** [comment]
Nice. Reminder for me/design to track what we catch here so we can double check Figma matches.

**Comment by ShubhamPalriwala** [comment]
@isabellewei I am unable to view the current references.
Would like to work on this

**Comment by isabellewei** [comment]
@ShubhamPalriwala The link is to the cLabs internal Slack. I've updated this issue with the info in the link :)

**Comment by ShubhamPalriwala** [comment]
Thanks @isabellewei,
Can I resolve this issue by changing it to `verify` or `verification` throughout the repository ?

**Comment by isabellewei** [comment]
yep! All text displayed in Valora is stored in `packages/mobile/locales`, and you can just edit the json files in there.

**Comment by ShubhamPalriwala** [comment]
@isabellewei Please review the PR !
